Custom Signup Route Backend

Tridib2510 opened the pull request:
Please check out this pull request.
I have implemented the Sign up route, added the database and ensure proper encryption and validation of email and phone.

Tridib2510 committed:
Add a custom signup route in the backend

Tridib2510 committed:
Add a custom signup route in the backend

AnkanMisra commented:
@Itz-snj Will look into the commit and will take on the PR from here

Itz-snj commented:
> Please check out this pull request. I have implemented the Sign up route, added the database and ensure proper encryption and validation of email and phone.


The code's good to go!
Please do share the api end points testing routes for the backend in your PR description, also make sure to update the code in the apertre2.0 branch not in the main branch.

AnkanMisra commented:
@Tridib2510 
1. Please add the package-lock.json to the .gitignore file
2. I've created a new branch called 'aperture2.0' - please refer to this branch instead of main going forward

Tridib2510 commented:
The api end points are /api/v1/user/signUp.
I hope there are no more problems .

Tridib2510 commented:
> > Please check out this pull request. I have implemented the Sign up route, added the database and ensure proper encryption and validation of email and phone.
> 
> The code's good to go! Please do share the api end points testing routes for the backend in your PR description, also make sure to update the code in the apertre2.0 branch not in the main branch.

The api end points are /api/v1/user/signUp.
I hope there are no more problems .

AnkanMisra commented:
@Tridib2510 The branch is the problem mate
Change the contri branch from main to apertre2.0
